common/changes/xingwa-dropdown_2017-01-11-08-14.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Remove 'stopPropagation' if dropdown doesn't handle the key event.",
```

Comment:
Can you adjust this comment:

Dropdown: Avoid calling stopPropagation on pressing escape if the dropdown is not expanded.

Also it's a patch, not a minor change.